packages/admin/admin/src/hooks/useAsyncOptionsProps.ts

```diff
@@ -11,12 +11,14 @@ export interface AsyncOptionsProps<T> {
 export function useAsyncOptionsProps<T>(loadOptions: () => Promise<T[]>): AsyncOptionsProps<T> {
     const [open, setOpen] = useState(false);
     const [options, setOptions] = useState<T[]>([]);
-    const loading = open && options.length === 0;
+    const [loading, setLoading] = useState(false);
```

Comment:
I believe we once had this loading state and intentionally reverted it, but I can't remember why. Maybe we'll find out in the future.

Reply:
mhm,... in my opinion we somehow must track the loading state.